packages/@aws-cdk/aws-batch/test/job-definition.test.ts

```diff
@@ -0,0 +1,118 @@
+import '@aws-cdk/assert/jest';
+import { ResourcePart } from '@aws-cdk/assert/lib/assertions/have-resource';
+import { InstanceClass, InstanceSize, InstanceType } from '@aws-cdk/aws-ec2';
+import { Repository } from '@aws-cdk/aws-ecr';
+import { EcrImage, LinuxParameters, MountPoint, Ulimit, Volume } from '@aws-cdk/aws-ecs';
+import * as iam from '@aws-cdk/aws-iam';
```

Comment:
We use `import iam = require('@aws-cdk/aws-iam`);`, would be great to be consistent.

Reply:
Got it 👍